### DISCUSSION TITLE
User folder location in Linux

### DISCUSSION THREAD

**Discussion by MilesSummers** [DISTRHO/Cardinal#678] [Q&A] [answered]
The User folder defaults to my $HOME directory.  This means that I am getting all sorts of directories for various plugins created in my home directory (not ideal).  Is there some way to change the User folder? I could find nothing in ~/.config/Cardinal/main.json to allow me to change it.

**Comment by dromer** [comment, marked as answer]
This is mostly hardcoded here: https://github.com/DISTRHO/Cardinal/blob/main/src/CardinalCommon.cpp#L604-L658

There is some minor control through `$XDG_DOCUMENTS_DIR` environment variable.
